packages/roosterjs-editor-plugins/lib/plugins/Paste/sourceValidations/constants.ts

```diff
@@ -0,0 +1,2 @@
+export const GOOGLE_SHEET_NODE_NAME = 'google-sheets-html-origin';
```

Comment:
Add comment with "@internal"

Not sure why the dts tool not fail here.

Same for other internal exports.